package.json

```diff
@@ -19,6 +19,7 @@
   "dependencies": {
     "ember-cli": "^5.11.0",
     "lodash": "^4.17.21",
+    "resolve-bin": "^1.0.1",
```

Comment:
is this only used in tests? 🤔 shouldn't it go in devDeps then?

Reply:
yes!, thank you